command/push_test.go

```diff
@@ -123,15 +121,18 @@ func TestPush_goodBackendInit(t *testing.T) {
 		// Expected weird behavior, doesn't affect unpackaging
 		".terraform/",
 		".terraform/",
-		".terraform/plugins/",
-		fmt.Sprintf(".terraform/plugins/%s_%s/", runtime.GOOS, runtime.GOARCH),
-		fmt.Sprintf(".terraform/plugins/%s_%s/lock.json", runtime.GOOS, runtime.GOARCH),
+
+		// this comnfig contains no plugins
```

Comment:
If you're going to leave code commented out here, please put a comment above it explaining why it's commented out and what would need to be done for it to be un-commented in future.